Automattic/newspack-plugin
# Fix/campaign filters

### All Submissions:






### Changes proposed in this Pull Request:

Addresses two issues with Campaigns Wizard filters. First, as described in https://github.com/Automattic/newspack-plugin/issues/759, when switching between tabs the filters will display the options for the previous tab. Second, when switching tabs if the last filter setting doesn't exist in the new tab, it will reset to "All."

Closes https://github.com/Automattic/newspack-plugin/issues/759

### How to test the changes in this Pull Request:

1. On `master`, go through the repro flow described in https://github.com/Automattic/newspack-plugin/issues/759. Observe the issue with incorrect filter values.
2. Switch to this branch, observe the issue is corrected.
3. In one tab select a filter that does not exist in the other, then switch back to the other. Observe the filter resets to "All."
4. Create a collection of campaigns of different types and statuses, and verify that the wizard works as expected.

### Other information:
